src/components/ui/Switch/stories/Switch.stories.tsx

```
@@ -1,18 +1,19 @@
-import { useState } from 'react';
-import Switch from '../Switch';
+import { JSX, useState } from 'react';
```

ESLint

[error] '/home/jailuser/git/node_modules/react/index.js' imported multiple times.

(import/no-duplicates)